hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/info/NetworkInfo.java

```diff
@@ -5,22 +5,30 @@
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- *      http://www.apache.org/licenses/LICENSE-2.0
+ *        http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
+ *
  */
 
-package com.hedera.node.app.spi.meta;
+package com.hedera.node.app.spi.info;
 
 import com.hedera.pbj.runtime.io.buffer.Bytes;
 
 /**
- * Provides context for query processing.
+ * Provides information about the network.
  */
-public interface QueryContext {
-    Bytes getLedgerId();
+public interface NetworkInfo {
```

Comment:
FunctionalInterface

Reply:
It is the first version of a facility which will get more methods later.